content/articles/pointers-in-c#/index.md.md

```diff
@@ -0,0 +1,244 @@
+### Getting started
+A pointer is a variable in the C# that stores the memory address of another variable.
+
+In C#, data types are categorized into three categories: `pointer types`, `value types`, and `reference types`. The memory address of another kind is stored in a pointer type variable. Since it lacks an individual variable, the reference data type holds a reference to another data type. Data types are explicitly included within the value data form. Integers, characters, and floating-point numbers, as well as the alphabet and numbers, are examples of value data types.
+
+### Making a pointer style declaration
+In C#, we declare pointers as illustrated below:
+
+```c#
+type *variable_name;
+```
+
+Where `*` is called the de-reference administrator. The de-reference administrator or de-reference operator is used for getting the value from the address that the pointer refers to.
+
+Consider the following example: 
+
+```c#
+int w=76
+int *ptr =&w;
+Console.WriteLine((int)ptr) 
+Console.WriteLine(*ptr)     .
+```
+
+`w` is a pointer variable that can be used to hold the position of an `int` sort. The operator `&` is known as a reference operator, and it is used to obtain a variable address. The memory address of the variable `w`, which can be assigned to a pointer variable, is specified by the symbols `&w`.
+
+### Safe and unsafe codes
+**Safe codes** are C# keywords that run under the `Common Language Runtime's supervision (CLR)` while **Unsafe codes** are C# keywords that execute outside the management of the CLR. Unlike C++ and C programming languages which use safe codes with pointers, the C# programming language only allows the use of unsafe codes. The unsafe codes may be used as a modifier or to label a group of statements as unsafe. Common language Runtime translates safe codes into software instructions, which are then executed by the computer's CPU.
+
+The example below uses the unsafe codes:
+
+```C#
+using System;
+namespace UnsafeCodeApplication
+
+{
+    class Demo
+    {
+        public void Method()
+        {
+            unsafe
+            {
+                int m = 60;
+                int n = 30;
+                int* ptr1 = &m;
+                int* ptr2 = &n;
+                Console.WriteLine(*ptr1);    
+                Console.WriteLine(*ptr2);    
+                Console.WriteLine((int)ptr1); 
+                Console.WriteLine((int)ptr2); 
+            }
+        }
+    }
+    class Example
+    {
+        public static void Main()
+        {
+            Demo d = new Demo();
+            d.Method();
+        }
+    }
+} 
+```
+
+![Output](/engineering-education/active-learning/unsafe%20codes.PNG)
```

Comment:
The folder name is wrong. And no need for the image, please provide the output as a text.